src/Wizard/Wizard.stories.tsx

```diff
@@ -42,11 +44,45 @@ export const Default = {
   render: Template
 };
 
-export const WithTitle = {
+export const WithTitleTypeString = {
   args: {
     ...Default.args,
     title: "Wizard Title"
   },
 
   render: Template
 };
+
+export const WithTitleTypeElement = {
+  args: {
+    ...Default.args,
+    title: (
+      <Stack
+        sx={{
+          alignItems: "center",
+          display: "flex",
+          flexDirection: "row",
+          gap: 1,
+          maxWidth: theme => theme?.layout?.content?.maxWidth,
```

Comment:
I get a type-error from IntelliSense (but not from running `npm run type-check`), could you please take a look?
![image (2)](https://github.com/user-attachments/assets/7cb24248-587f-4fc3-94fb-55a4844c0d3d)

In case you manage to fix it, it also happens here (this was already in the code before your modifications):  https://github.com/IPG-Automotive-UK/react-ui/blob/b547cf47dc0c794fc75850b3bf4399967e539ef3/src/Wizard/Wizard.tsx#L27

Reply:
Couldnt find a solution and not wasting much time here. Informed @lukemojo about this, i think he will investigate it seperatily.